### ISSUE TITLE
several bots configuration

### ISSUE THREAD

**Issue by hafezdivandari** [irazasyed/telegram-bot-sdk#370]
Hi, I have several Bot in my application but there is only one set of configurations in the config file.
I think it should be possible to set multiple bots, like what we do when [Using Multiple Database Connections](https://laravel.com/docs/5.4/database#using-multiple-database-connections)

```
return [

    /*
    |--------------------------------------------------------------------------
    | Default Bot
    |--------------------------------------------------------------------------
    |
    */

    'default' => 'bot1',

    /*
    |--------------------------------------------------------------------------
    | Bot Connections
    |--------------------------------------------------------------------------
    |
    */

    'connections' => [

        'bot1' => [
            'bot_token' => env('TELEGRAM_BOT_TOKEN', '1:eff'),
            'async_requests' => env('TELEGRAM_ASYNC_REQUESTS', true),
            'http_client_handler' => null,
            'commands' => [
                    Telegram\Bot\Commands\HelpCommand::class,
                ],
        ],

        'bot2' => [
            'bot_token' => env('TELEGRAM_BOT_TOKEN', '2:eff'),
            'async_requests' => env('TELEGRAM_ASYNC_REQUESTS', false),
            'http_client_handler' => null,
            'commands' => [
                    Telegram\Bot\Commands\HelpCommand::class,
                ],
        ],

        'bot3' => [
            'bot_token' => env('TELEGRAM_BOT_TOKEN', '3:eff'),
            'async_requests' => env('TELEGRAM_ASYNC_REQUESTS', true),
            'http_client_handler' => null,
            'commands' => [
                    Telegram\Bot\Commands\HelpCommand::class,
                ],
        ],

    ],
```

**Comment by irazasyed** [comment]
Hi, Yes! Multi-bot is supported.

You can work on it using `Telegram::bot('bot1')->sendMessage()` like so. Where `bot1` is your keyname of the config. 

Your webhook should be dynamic. Something like `/bot/{bot}/secret/webhook` and then pass the `{bot}` value to `Telegram::bot($bot)->getWebhookUpdate()` and so on.

Let me know if you have any questions. All the methods work the same as it works for a single bot. And ofc the commands can be either shared or registered separately for each of them.

**Comment by hafezdivandari** [comment]
how about config file? How should I write it? was my sample correct?

**Comment by irazasyed** [comment]
You can refer the [config](https://github.com/irazasyed/telegram-bot-sdk/blob/develop/src/Laravel/config/telegram.php) file. Obviously, you need to use the v3 (unstable) or `dev-develop` branch + publish the config file (If on Laravel).

**Comment by timbogdanov** [comment]
@irazasyed does CommandHandler not work when running multiple bots? I have 2 bots that retrieve updates with the commands handler, but it seems to be sending responses to the wrong bots

```php
$update = Telegram::bot('bot1')->commandsHandler(true);
```

I should also mention that I have set up unique web-hooks for both bots.

When i send a /start command to one both, it returns the start message successfully. But when i send it to the second bot, to web-hook update comes through, but it doesn't handle the command, and returns nothing


`Update:` I fiddled around, now no matter what bot i do the /start command, it always sends the same start response message to one of the bots.

`Update 2:` I fiddled around more, I assume `php artisan cache:clear` did the trick, or just a general `php artisan optimize`
